.circleci/config.yml

```diff
@@ -1,19 +1,32 @@
 version: 2
 jobs:
   build:
-    docker:
-      - image: circleci/python:2.7-stretch
+    machine:
+      image: circleci/classic:latest
+    environment:
+      GOOGLE_PROJECT_ID: ci-test-data-explorer
     steps:
       - checkout
       - restore_cache:
           key: virtualenv-{{ .Branch }}-{{ checksum "bigquery/requirements.txt" }}
       - run:
           name: Run the yapf python linter
           command: |
-            virtualenv ~/yapf
-            source ~/yapf/bin/activate
             pip install yapf
             yapf -dr .
+      - run:
+          name: Run the BigQuery Indexer
+          command: |
+            echo ${GOOGLE_SERVICE_KEY} | base64 --decode > ${HOME}/.config/gcloud/application_default_credentials.json
+            cd bigquery
+            docker network create dataexplorer_default
```

Comment:
nit switch lines 21 and 22. The echo line and docker network line are one-time initialization, and won't be needed for other indexers.

Reply:
Done